src/deployment/windows.md

```diff
@@ -10,117 +10,152 @@ Flutter app to the [Microsoft Store][microsoftstore].
 ## Preliminaries
 
 Before beginning the process of releasing 
-a Flutter Windows Desktop app, it is necessary
-to confirm that it satisfies [Microsoft Store Policies][storepolicies].
+a Flutter Windows desktop app, first confirm
+that it satisfies [Microsoft Store Policies][storepolicies].
 
-Joining the [Microsoft Partner Network][microsoftpartner] is also required.
+Also, join the
```

Comment:
This sounds like an endorsement, versus the old text which just called it out as a prerequisite.

Reply:
Changed to "You must join the..."